raise Exception("This is a custom exception message")
